README.md

```diff
@@ -1,10 +1,8 @@
-### Valkey GLIDE
-
-Valkey General Language Independent Driver for the Enterprise (GLIDE), is an AWS-sponsored, open-source Valkey client that includes support for open-source Redis 6.2 to 7.2. Valkey GLIDE works with any distribution that adheres to the Redis Serialization Protocol (RESP) specification, including Amazon ElastiCache, and Amazon MemoryDB.
-Strategic, mission-critical applications have requirements for security, optimized performance, minimal downtime, and observability. Valkey GLIDE is designed to provide a client experience that helps meet these objectives. It is sponsored and supported by AWS, and comes pre-configured with best practices learned from over a decade of operating RESP-compatible services used by hundreds of thousands of customers. To help ensure consistency in development and operations, Valkey GLIDE is implemented using a core driver framework, written in Rust, with extensions made available for each supported programming language. This design ensures that updates easily propagate to each language and reduces overall complexity. In this Preview release, Valkey GLIDE is available for Python and Java, with support for Javascript (Node.js) actively under development.
+# Valkey GLIDE
+Valkey General Language Independent Driver for the Enterprise (GLIDE), is an AWS-sponsored, open-source Valkey client that includes support for open-source Redis 6.2 to 7.2. Valkey GLIDE works with any distribution that adheres to the Redis Serialization Protocol (RESP) specification. Strategic, mission-critical applications have requirements for security, optimized performance, minimal downtime, and observability. Valkey GLIDE is designed to provide a client experience that helps meet these objectives. It comes pre-configured with best practices learned from over a decade of operating RESP-compatible services. To help ensure consistency in development and operations, Valkey GLIDE is implemented using a core driver framework, written in Rust, with extensions available for each supported programming language. This design ensures that updates easily propagate to each language and reduces overall complexity. 
```

Comment:
```suggestion
Valkey General Language Independent Driver for the Enterprise (GLIDE) is an open-source Valkey client that includes support for open-source Valkey and Redis servers. Valkey GLIDE works with any distribution that adheres to the Redis Serialization Protocol (RESP) specification. Strategic, mission-critical applications have requirements for security, optimized performance, minimal downtime, and observability. Valkey GLIDE is designed to provide a client experience that helps meet these objectives. Sponsored and supported by AWS, it comes pre-configured with best practices learned from over a decade of operating RESP-compatible services. To help ensure consistency in development and operations, Valkey GLIDE is implemented using a core driver framework, written in Rust, with binding available for a wide range of popular programming languages. This design ensures that updates easily propagate to each language and reduces overall complexity.
```

Reply:
This part should be discussed with Mickey and we should copy it from the blog